src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,30 @@
+import { Todo } from '../../types/Todo';
```

Comment:
It is recommended to add 'React' import statement since you're using 'React.FC' on line 11.

Reply:
updated